admin-ui/src/component/Unit/NewSpaceModal.tsx

```diff
@@ -284,21 +299,25 @@ function FirstPage({
           max={10}
           required
         />
-        <br />
-        <Select
-          id="parent"
-          label={t("SpaceModal.page1.parentLabel")}
-          placeholder={t("SpaceModal.page1.parentPlaceholder")}
-          required
-          helper={t("SpaceModal.page1.parentHelperText")}
-          options={parentOptions}
-          onChange={(selected: OptionType) =>
-            dispatch({
-              type: "setParentSpace",
-              parentSpace: selected.value as string,
-            })
-          }
-        />
+        {parentSpace == null ? (
```

Comment:
Not using `isRootSpace` here makes me wonder if there's some subtle logic here I don't quite get (non-strict equality operator).

Reply:
thanks, I refactored this to a hopefully more clear form